javascript/src/controllers/file-size-limit-controller.js

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus'
+
+export class FileSizeLimitController extends Controller {
```

Comment:
Creo que un mejor nombre sería:

```suggestion
export class TrixAttachmentsLimitController extends Controller {
```

porque todo esto es específico al editor de Trix, esto no lo podrías usar en otro contexto